Cargo.toml

```diff
@@ -32,11 +32,19 @@ thiserror = "1.0"
 askama = "0.10"
 
 # This only supports up to 0.7. See examples/actix/main.rs for more.
-actix = "0.9"
+actix = "0.10"
 actix-web = { version = "2.0", features = ["openssl"] }
 actix-session = "0.3"
 actix-files = "0.2"
 
+# Example, use the version numbers you need
+tide = "0.15.0"
+tide-rustls = "0.1.3"
+askama_tide = "0.12.0"
+async-std = { version = "1.6.0", features = ["attributes"] }
+xactor = "0.7.8"
```

Comment:
I think it would be good here to remove xactor and actix completely if we are going to jump to tide :)